### PR TITLE
Fix bugs in partner detail view

### PR DESCRIPTION
**Pull request by suecarmol**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Fix several bugs in partner detail view.
- Add margin to `List applications` button.
- Display formatting for tags and languages
- Fix margins that hid timeline bubbles
- Fix image display

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This makes the partner detail view more useable

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T296900](https://phabricator.wikimedia.org/T296900)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
Tested manually

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/static/css/new-local.css [hunk 1]
```diff
@@ -1345,6 +1345,31 @@ ul.timeline-home li {
...
+    min-height: 3em;
+    z-index: 1;
+}
+
+.resource-label-subject{
+    background-image: url('../img/bullet-subject.png');
+    background-repeat: no-repeat;
+    background-position: top left;
+    min-height: 2em;
+    padding: 6px 0px 0px 25px;
```
Comment (jsnshrmn) [inline TWLight/static/css/new-local.css:1362]: For me, the background image for this and the languages butt up against the text
image

increasing that padding from 25 to 30 px helps out.
image

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
I have one small change request, but otherwise this is looking good to me.

File: TWLight/static/css/new-local.css (hunk 1 above)
Comment (suecarmol) [reply TWLight/static/css/new-local.css]: Good catch!

**Review by jsnshrmn — APPROVED** [review]

**Comment by jsnshrmn** [comment]
This looks good to me, but Sam had some additional feedback this morning on the phab ticket

**Pull request merged**